pages/services/prometheus/0.1.0-2.3.1/install/install.md

```diff
@@ -0,0 +1,299 @@
+
+## Installing HA-global prometheus 
+
+To federate data from slave prometheus to global prometheus,HA-global prometheus is requird to be installed.
+To install global prometheus, you would require to mention global prometheus specific configuration changes to prometheus yml and slave prometheus end points needs to be passed as target with in single quotes comma separated.
+
+Note: mention global prometheus configuration only when you require federation else its not recommended to be checked.
+
+## Install HA-Prometheus standalone with no linkage to Alert Manager\Global Prometheus:
+
+To Install HA-Prometheus server without alert manager\global prometheus , you should not check on alert manager check box ,  and should not put global prometheus configuration, alert manager target under prometheus yml.
+
+With these configuration HA-Prometheus server would be launched without pointing to HA-Alert Manager.
+
+## Installing multiple instances
+
+By default, the prometheus is installed with a service name of prometheus. You may specify a different name using a custom service configuration as follows:
+
+   ```shell
+   {
+       "service": {
+           "name": "prometheus-other"
+       }
+   }
+   ```
+
+When the above JSON configuration is passed to the package install prometheus  command via the --options argument, the new service will use the name specified in that JSON configuration:
+
+   ```shell
+   dcos package install prometheus --options=prometheus-other.json
+   ```
+   
+Multiple instances of Prometheus may be installed into your DC/OS cluster by customizing the name of each instance. For example, you might have one instance of Prometheus named prometheus-staging and another named prometheus-prod, each with its own custom  configuration.
+
+After specifying a custom name for your instance, it can be reached using dcos prometheus CLI commands or directly over HTTP as described below.
+
+Note: The service name cannot be changed after initial install. Changing the service name would require installing a new instance of the service against the new name, then copying over any data as necessary to the new instance.
+
+## Installing into folders
+
+In DC/OS 1.10 and above, services may be installed into folders by specifying a slash-delimited service name. For example:
+
+   ```shell
+   {
+       "service": {
+           "name": "/foldered/path/to/prometheus"
+       }
+   }
+   ```
+The above example will install the service under a path of foldered => path => to => prometheus. It can then be reached using dcos prometheus  CLI commands or directly over HTTP as described below.
+
+Note:  The service folder location cannot be changed after initial install.Changing the service location would require installing a new instance of the service against the new location, then copying over any data as necessary to the new instance.
+
+## Service Discovery
```

Comment:
The entire `Service Discovery section` needs revision. This is not what exists in the framework.

Reply:
Updated